scripts/cleanup.py

```diff
@@ -0,0 +1,13 @@
+import os
```

Comment:
Let's move this to app folder as this scripts works only on app media directory for now.

Reply:
Ok, I will update the PR for the same and will let you know once done